Sources/Hub/Hub.swift

```diff
@@ -202,7 +202,8 @@ public class LanguageModelConfigurationFromHub {
     }
 
     static func fallbackTokenizerConfig(for modelType: String) -> Config? {
-        guard let url = Bundle.module.url(forResource: "\(modelType)_tokenizer_config", withExtension: "json") else { return nil }
+        let bundle = Bundle(for: self)
```

Comment:
We could fix the tests doing something like:

```suggestion
        let bundle = NSClassFromString("XCTest") != nil ? Bundle.module : Bundle(for: self)
```

But upon additional testing I found that building a CLI that uses `Tokenizers` would not find the fallback configuration either.

Do you have any thoughts on that? Some additional ideas:
- Iterate through the bundles and select the right one.
- Create a bundle (or framework?) solely dedicated to hosting the resources.
- Download the fallback configs from a well-known location in the Hub.